src/resources/packages/classy/components/Classy.tsx

```diff
@@ -241,12 +215,18 @@ function ClassyApplication() {
 	);
 }
 
-export function Classy( { registry }: { registry: WPDataRegistry } ) {
+/**
+ * The Classy application rendered in the context of the Classy provider that will provide the application
+ * with access to the WordPress default registry.
+ *
+ * @since TBD
+ *
+ * @constructor
```

Comment:
Just curious for my own learning: When should we be using the `@constructor` doc tag in a JS/React context?